### ISSUE TITLE
Departure board SVG has useless stray points

### ISSUE THREAD

**Issue by davwheat** [google/material-design-icons#1828]
See the blue points in the image below.

![Image](https://github.com/user-attachments/assets/e58bfee2-1ca9-4c85-96e4-83067b3cbb07)

![Image](https://github.com/user-attachments/assets/ab5a0c21-2b03-40b4-82bf-872a15bfcef6)

This results in some rendering artifacts in Firefox

![Image](https://github.com/user-attachments/assets/82a6117a-98e1-4280-83fd-919d9c13ec04)

**Comment by tphinney** [comment]
Ah. OK, that is a bug. We should be hiding the points that need to expand into the Filled shape in the existing contours, precisely because Apple will render a zero-thickness line like what you see there. I can see in my icon source that we failed to do that.

Would I be right to guess you are viewing this on macOS or iOS? Not Windows or Linux.

At 100% fill we have a whole different workaround just to avoid this. We are super familiar with the issue and just in this case forgot that we have to avoid it in a different way at zero Fill. Oops. Rats!

Sorry about that. It will likely be at least a week or two before a fix percolates out to the repo and to the live font being served.

**Comment by davwheat** [comment]
> Ah. OK, that is a bug. We should be hiding the points that need to expand into the Filled shape in the existing contours, precisely because Apple will render a zero-thickness line like what you see there. I can see in my icon source that we failed to do that.
> 
> Would I be right to guess you are viewing this on macOS or iOS? Not Windows or Linux.

When I first saw it (which was where I took the browser screenshot) was on the MDI Google Font page in Firefox on Windows.

From later testing (now) it seems to not affect the _SVG_ on any of the browsers or devices I've tested on (I only run Linux and Windows), but _does_ affect Firefox and Chrome's (to a lesser extent) **font** rendering engines. Whether that's expected from the font I don't know! :)

**Firefox Windows:**
![Image](https://github.com/user-attachments/assets/aada5ea6-6cae-40a4-a72c-2d95a7119ad3)

**Chrome Windows:**
![Image](https://github.com/user-attachments/assets/e51d5448-60f4-488f-ae7c-c1d583987a91)

It's not an issue or blocker for me right now anyway -- I just deleted the points and continued with my day importing it into Android Studio. Thanks for following up so quickly!
